parm/postcntrl_gfs_two.xml

```diff
@@ -1874,6 +1879,11 @@
        <scale>3.0</scale>
     </param>
 
+    <param>
```

Comment:
@JesseMeng-NOAA Please remove SLR from flux data set and update the corresponding txt file.

Reply:
Done